### PR TITLE
Movie page

### PR DESCRIPTION
**Pull request by walmarek**
Please review!

### CODE REVIEW

File: src/core/getData.js
Comment (xxFino) [inline src/core/getData.js:1]: Good solution!

File: src/feature/Movie/MoviePage/MoviePage.js
Comment (xxFino) [inline src/feature/Movie/MoviePage/MoviePage.js:1]: Good job!

File: src/feature/Movie/MoviePage/MoviePageDetails/Backdrop/index.js
Comment (xxFino) [inline src/feature/Movie/MoviePage/MoviePageDetails/Backdrop/index.js:1]: Great work!

File: src/feature/Movie/MoviePage/MoviePageDetails/Backdrop/styled.js
Comment (xxFino) [inline src/feature/Movie/MoviePage/MoviePageDetails/Backdrop/styled.js:1]: Good job!

File: src/feature/Movie/MoviePage/moviePageSlice.js
Comment (xxFino) [inline src/feature/Movie/MoviePage/moviePageSlice.js:1]: Nice!

**Review by xxFino — APPROVED** [review]
A lot of great work! Good job

**Review by MariolaFilipiak — APPROVED** [review]

**Review by MariolaFilipiak — APPROVED** [review]
Nice! :)

**Review by MariolaFilipiak — APPROVED** [review]
i need do this everywere ;(

**Review by MariolaFilipiak — APPROVED** [review]
Looks good

**Review by MariolaFilipiak — APPROVED** [review]
Bravo!

**Review by MariolaFilipiak — APPROVED** [review]

File: src/feature/Movie/MoviePage/MoviePage.js
Comment (MariolaFilipiak) [reply src/feature/Movie/MoviePage/MoviePage.js:1]: I think so too

**Review by MariolaFilipiak — APPROVED** [review]
Good Job

**Review by MariolaFilipiak — APPROVED** [review]

**Pull request merged**